### PR TITLE
Fix/check

### PR DESCRIPTION
**Pull request by wilsonianb**

### CODE REVIEW

File: src/ripple_txt.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default class RippleTxt {
 
     return new Promise((resolve, reject) => {
       RippleVaultClient.RippleTxt.get(domain, (error, resp) => {
-        if (Object.keys(resp).length === 0 || !resp) {
+        if (!resp || Object.keys(resp).length === 0) {
```
Comment (stevenzeiler) [inline src/ripple_txt.js]: would there be an `error` object here if `!resp`?

Comment (wilsonianb) [inline src/ripple_txt.js]: Hmm, probably, I guess I should just move the check for `error` to the top

Comment (stevenzeiler) [inline src/ripple_txt.js]: The nodejs convention is to always define either `error` or `resp` but not both.

**Comment by wilsonianb** [comment]
Now checking for `error` instead of `!resp`

**Comment by stevenzeiler** [comment]
:+1:  LGTM

**Comment by wilsonianb** [comment]
:+1:

**Pull request merged**